jerry-core/jcontext/jcontext.c

```diff
@@ -19,6 +19,50 @@
  * @{
  */
 
+#ifdef JERRY_ENABLE_EXTERNAL_CONTEXT
+
+/**
+ * Get the current context from the current instance.
+ *
+ * @return the pointer to the context
+ */
+jerry_context_t *
+jerry_context_get_current_context (void)
```

Comment:
32-bit x86: --compile-flag=-m32
size of `jerry_get_user_context`
"current": 19 byte
"inline": 22 byte

ARMv7l: --toolchain=cmake/toolchain_linux_armv7l.cmake
"current": 18 byte
"inline": 22 byte